debugger/src/debug_adapter.rs

```diff
@@ -1489,7 +1558,73 @@ impl<R: Read, W: Write> DebugAdapter<R, W> {
             self.send_event("output", Some(event_body))
         } else {
             //DebugCAdapterType::CommandLine
-            println!("{}", msg.into());
+            println!("{}", message.into());
+            true
+        }
+    }
+
+    /// Send a custom "probe-rs-show-message" event to the MS DAP Client
+    /// The `severity` field can be one of "information", "warning", or "error"
+    pub fn show_message<S: Into<String>>(&mut self, severity: String, message: S) -> bool {
```

Comment:
Maybe use the Severity enum I suggested in another comment here :)

Reply:
```suggestion
    pub fn show_message(&mut self, severity: String, message: impl Into<String>) -> bool {
```

I think this is more readable for `Into`, `From`, `AsRef` and the likes and just very simple trait constraints in general :)

Reply:
I like it ... thanks :)